debian/base/Dockerfile.rpm

```diff
@@ -36,12 +38,13 @@ ENV PYTHON_PIP_VERSION="9.0.1"
 # Confluent
 ENV SCALA_VERSION="2.11"
 ENV CONFLUENT_MAJOR_VERSION="3"
-ENV CONFLUENT_MINOR_VERSION="2"
+ENV CONFLUENT_MINOR_VERSION="3"
 
-ENV CONFLUENT_PATCH_VERSION="1"
+ENV CONFLUENT_PATCH_VERSION="0"
 ENV CONFLUENT_VERSION="$CONFLUENT_MAJOR_VERSION.$CONFLUENT_MINOR_VERSION.$CONFLUENT_PATCH_VERSION"
 
-ENV KAFKA_VERSION="0.10.2.1"
+# Add -0.1.SNAPSHOT for internal releases.
+ENV KAFKA_VERSION="0.11.0.0-0.1.SNAPSHOT"
```

Comment:
Some things like this may be a bit hard to script. Oh well. For anything that we can't automate, let's add to a README though so someone can go thru a list next time.